azure-core/README.md

```diff
@@ -0,0 +1,357 @@
+
+
+# Azure Core Library
+
+## Pipeline
+
+The Azure Core pipeline is a re-strucuting of the msrest pipeline introduced in msrest 0.6.0.
+Further discussions on the msrest implementation can be found in the [msrest wiki](https://github.com/Azure/msrest-for-python/wiki/msrest-0.6.0---Pipeline).
+
+The Azure Core Pipeline is an implementation of chained policies as described in the [Azure SDK guidelines](https://github.com/Azure/azure-sdk/tree/master/docs/design).
+
+The Python implementation of the pipeline has some mechanisms specific to Python. This is due to the fact that both synchronous and asynchronous implementations of the pipeline must be supported indepedently.
+
+When constructing an SDK, a developer may consume the pipeline like so:
+
+```python
+from azure.core import Configuration, Pipeline
+from azure.core.transport import RequestsTransport, HttpRequest
+from azure.core.pipeline.policies import (
+    UserAgentPolicy,
+    HeadersPolicy,
+    RetryPolicy,
+    RedirectPolicy,
+    ContentDecodePolicy
+)
+
+class FooServiceClient():
+
+    @staticmethod
+    def create_config(**kwargs):
+        # Here the SDK developer would define the default
+        # config to interact with the service
+        config = Configuration(**kwargs)
+        config.user_agent_policy = UserAgentPolicy("ServiceUserAgentValue", **kwargs)
+        config.headers_policy = HeadersPolicy({"CustomHeader": "Value"})
+        config.retry_policy = RetryPolicy(**kwargs)
+        config.redirect_policy = RedirectPolicy(**kwargs)
+        config.logging_policy = NetworkTraceLoggingPolicy(**kwargs)
```

Comment:
I believe we either need a separate Logging and NetworkTraceLoggingPolicy instances, or we need to be able to configure the NetworkTraceLoggingPolicy to include or not include the payload. 

In short, I'd expect the request line/response status and response (possibly with duration) as Info level logs, and if (and only if) I have debug level tracing and passed in the http network log flag, we include the bodies.

Reply:
I think what you describe could exist as additional configuration options in the NetworkTraceLoggingPolicy

Reply:
Then we may consider renaming the policy. Ultimately, I'd expect we have a similarly named policy for all the languages. And I believe that LoggingPolicy is used today. I'm fine with opening an issue tracking the fact that we need to standardize the name...

Reply:
Yup, we can go with LoggingPolicy.